Mathlib/Data/Finset/Max.lean

```diff
@@ -251,6 +251,20 @@ lemma min'_eq_iff (a : α) : s.min' H = a ↔ a ∈ s ∧ ∀ (b : α), b ∈ s
 lemma max'_eq_iff (a : α) : s.max' H = a ↔ a ∈ s ∧ ∀ (b : α), b ∈ s → b ≤ a :=
   ⟨(· ▸ ⟨max'_mem _ _, le_max' _⟩), fun h ↦ le_antisymm (max'_le _ _ _ h.2) (le_max' _ _ h.1)⟩
 
+@[simp] lemma min'_pair (a b : α) : min' {a, b} (insert_nonempty _ _) = min a b := by
+  rw [Finset.min'_eq_iff]
+  simp
+  rcases le_or_gt a b with h | h
+  · exact .inl h
+  · exact .inr h.le
```

Comment:
If you move this down the file the proof is just
```suggestion
  rw [Finset.min'_insert, Finset.min'_singleton, min_comm]
```

Reply:
The `min_comm` suggests to me that `min'_insert` might be stated backwards, though I haven't thought much about it.